src/Sylius/Bundle/ThemeBundle/Translation/DependencyInjection/Compiler/TranslatorFallbackLocalesPass.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class TranslatorFallbackLocalesPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        try {
+            $symfonyTranslator = $container->findDefinition('translator.default');
```

Comment:
[Symfony documentation](http://symfony.com/doc/current/components/dependency_injection/compilation.html#prepending-configuration-passed-to-the-extension):

> Make sure your compiler pass does not require services to exist. Abort the method call if some required service is not available.

And `matthiasnoback/symfony-dependency-injection-test` [encourages to keep up with this practice](https://github.com/matthiasnoback/SymfonyDependencyInjectionTest#standard-test-for-unobtrusiveness):

> The AbstractCompilerPassTestCase class always executes one specific test - compilation_should_not_fail_with_empty_container() - which makes sure that the compiler pass is unobtrusive. For example, when your compiler pass assumes the existence of a service, an exception will be thrown, and this test will fail